include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -5050,7 +5303,7 @@ struct OrtCustomOp {
  *
  * See onnxruntime/test/shared_lib/test_model_editor_api.cc for example usage.
  *
- * \since Version 1.21.
+ * \since Version 1.22.
```

Comment:
The Model Editor API wasn't included in 1.21 so updating this value for the types/functions involved to make them correct.